packages/jira-adapter/src/filters/assets/assets_instances_addition.ts

```diff
@@ -44,6 +44,16 @@ const filter: FilterCreator = ({ config }) => ({
       .forEach(instance => {
         instance.value.objectSchemaId = getParent(instance).value.id
       })
+
+    await awu(changes)
+      .filter(isInstanceChange)
+      .map(getChangeData)
```

Comment:
I think you should add: 
```suggestion
       .filter(isAdditionChange)
      .map(getChangeData)
```

Reply:
its true for modification change as well but I added it for those.

Reply:
so maybe you should add it in a different filter, or change the filter's name

Reply:
you are correct. I changed the filter's name :)
Trying to reduce amount of filters.